keras/distribution/distribution_lib.py

```diff
@@ -327,10 +327,10 @@ class ModelParallel(Distribution):
     # will be split across 4 devices. Any other variable that doesn't
     # match any key in the layout map will be fully replicated.
     layout_map = LayoutMap(device_mesh)
-    layout_map['.*dense.*kernel'] = (None, 'model')
-    layout_map['.*dense.*bias'] = ('model',)
-    layout_map['.*conv2d.*kernel'] = (None, None, None, 'model')
-    layout_map['.*conv2d.*bias'] = ('model',)
+    layout_map['dense.*kernel'] = (None, 'model')
```

Comment:
what's the reason for `.*` in the middle instead of `/` again? the optimizer layout paths?

Reply:
oh, that's for dense_1/dense_2.

Reply:
ah right thanks! so i guess that will be up to the user's practice. if they are setting the name themselves on all layers, that won't be necessary (I think?), if they are not, they should leave that in

Reply:
Correct.